.github/dependabot.yml

```diff
@@ -13,3 +13,5 @@ updates:
       - "merge safe"
       - "target: patch"
     open-pull-requests-limit: 5
+    ignore:
+      - dependency-name: "@material/*"
```

Comment:
Should we also have `material-components-web`?

Reply:
Yeah doesn't hurt to add that in